bubblegum/program/src/utils.rs

```diff
@@ -67,15 +68,15 @@ pub fn replace_leaf<'info>(
     let authority_pda_signer = &[&seeds[..]];
     let cpi_ctx = CpiContext::new_with_signer(
         gummyroll_program.clone(),
-        gummyroll::cpi::accounts::Modify {
+        spl_compression::cpi::accounts::Modify {
             authority: authority.clone(),
-            merkle_roll: merkle_roll.clone(),
-            candy_wrapper: candy_wrapper.clone(),
+            merkle_tree: merkle_roll.clone(),
```

Comment:
Should all instances (such as this) of `merkle_roll` as a variable name be changed to merkle_tree now?

Similarly, is `merkle_slab` still the right name where it is used?